packages/cli/e2e-tests/test-project/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "root",
```

Comment:
Let's slim down all of the fixture files here to the bare minimum required for things to run, nothing extra. Let's also move it into a `__fixtures__` folder to make it clear what it is.

Reply:
Pared it down to just the front end, and just a single div for the front end. 

On a sidenote, was taking a look at the webpack debug stats and it's still taking 2 mins to compile. This is about the same amount of time as the whole example-app. The actual difference is about 12 seconds on my M1. The bootstrapping seems extreme given the relatively small amount of modules going in and code being output.

Any idea why this might be happening? 

Most of the time is spent in the `LOG from webpack.Compiler
<t> make hook: 131400.979833 ms`. The pared down version isn't using any modules other than 
```
import '@backstage/cli/asset-types';
import React from 'react';
import ReactDOM from 'react-dom';
```